optuna/pruners/_percentile.py

```diff
@@ -199,3 +194,16 @@ def prune(self, study: "optuna.study.Study", trial: "optuna.trial.FrozenTrial")
         if direction == StudyDirection.MAXIMIZE:
             return best_intermediate_result < p
         return best_intermediate_result > p
+
+    def is_target_step(self, step: int, trial: "optuna.trial.FrozenTrial") -> bool:
```

Comment:
To call `_is_first_in_interval_step` in this function, I need to add `trial` as an argument.

Reply:
is_prunable_step should be better.